Pspcl/Pspcl.Services/StockService.cs

```diff
@@ -506,8 +522,83 @@ public void UpdateStockMaterial(List<List<int>> selectedRowsToDelete)
             }
             _dbcontext.SaveChanges();
             
-        }       
+        }
+
+        public string UploadImageToAzure(IFormFile file)
+        {
+            string fileExtension = Path.GetExtension(file.FileName);
+            using MemoryStream fileUploadStream = new MemoryStream();
+            {
+                file.CopyTo(fileUploadStream);
+                fileUploadStream.Position = 0;
+                BlobContainerClient blobContainerClient = new BlobContainerClient(
+                    _azureOptions.ConnectionString,
+                    _azureOptions.Container);
+
+                var uniqueName = Guid.NewGuid().ToString() + fileExtension;
+
+                try
+                {
+                    //azure package is required for below line
+                    BlobClient blobClient = blobContainerClient.GetBlobClient(uniqueName);
+                    blobClient.Upload(fileUploadStream, new BlobUploadOptions()
+                    {
+                        HttpHeaders = new BlobHttpHeaders
+                        {
+                            ContentType = "image/bitmap"
```

Comment:
Are you sure that for each type of file a user can update, will have the same content type here?

Reply:
the user can upload files with different extensions, such as .jpg, .png .
so, for that should I need to add ' switch cases'? switch cases, that would get the file extension, and return me the content type dynamically? 
but the thing is, user will download the image anyhow, so once the file is downloaded, it is displayed exactly the same, so does specifying the content type makes big difference?

Reply:
The line on which I commented is from upload process, not download. So, talking about upload only. So, 2 things needs to be taken care here:
1. Proper validation of file types we are allowing or not allowing (No matter where we are doing that)
2. If user upload a picture which doesnt support the type you have hard coded in the above code, what would be the end result ?